code/modules/admin/admin.dm

```diff
@@ -709,11 +709,11 @@ ADMIN_VERB_ADD(/datum/admins/proc/delay, R_SERVER, FALSE)
 
 	if(!check_rights(R_SERVER))
 		return
-	if (SSticker.current_state != GAME_STATE_PREGAME)
+	if (SSticker.current_state != GAME_STATE_PREGAME && SSticker.current_state != GAME_STATE_STARTUP)
```

Comment:
this looks like its delaying the round start, not the end? Are you sure this delays the end of the round?

Reply:
This button might be pressed before full lobby initialization (before `STATE_STARTUP`) but in that case it will delay round end before game actually starts (but not lobby delay as it should be).